### PR TITLE
Updated module terraform-aws-s3-log-storage tag

### PR DESCRIPTION
**Pull request by MartinCanovas**
Updated module terraform-aws-s3-log-storage tag to 0.14.0. The tag 0.13.1 causes error with Terraform 0.13:

```
Error: Failed to query available provider packages
Could not retrieve the list of available versions for provider hashicorp/aws:
no available releases match the given constraints >= 3.0.*, >= 2.0.*, >=
2.0.*, >= 2.0.*, >= 2.0.*, >= 2.0.*, < 4.0.*, ~> 2.0, ~> 2.0, ~> 2.0, ~> 2.0,
~> 2.0, ~> 2.0, >= 2.0.*, < 4.0.*, >= 3.0.*, >= 2.0.*, >= 2.0.*, >= 2.0.*
```

## what
* Describe high-level what changed as a result of these commits (i.e. in plain-english, what do these changes mean?)
* Use bullet points to be concise and to the point.

## why
* Provide the justifications for the changes (e.g. business case). 
* Describe why these changes were made (e.g. why do these commits fix the problem?)
* Use bullet points to be concise and to the point.

## references
* Link to any supporting github issues or helpful documentation to add some context (e.g. stackoverflow). 
* Use `closes #123`, if this PR closes a GitHub issue `#123`

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -2,8 +2,8 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```
Comment (woz5999) [inline versions.tf:2]: can you remove the upper version pinning here too?

**Review by woz5999 — CHANGES_REQUESTED** [review]

File: versions.tf (hunk 1 above)
Comment (MartinCanovas) [reply versions.tf]: @woz5999 upper version limit removed.

**Review by woz5999 — DISMISSED** [review]

**Review by woz5999 — APPROVED** [review]

**Review by woz5999 — CHANGES_REQUESTED** [review]
@MartinCanovas It looks like the new s3 module you're upgrading too has some breaking changes that require additional updates here. Specifically, the `region` variable was removed to support aws versions 2.x and 3.x. 

Here are the relevant changes: https://github.com/cloudposse/terraform-aws-s3-log-storage/commit/32cfca282792756f0a9613e4bd4f7e3bf6484aca#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4

Can you update your PR to account for these changes and also updates the examples accordingly?

**Comment by MartinCanovas** [comment]
> @MartinCanovas It looks like the new s3 module you're upgrading too has some breaking changes that require additional updates here. Specifically, the `region` variable was removed to support aws versions 2.x and 3.x. Other variables were also removed to support the new `context.tf` pattern.
> 
> Here are the relevant changes: [cloudposse/terraform-aws-s3-log-storage@32cfca2#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4](https://github.com/cloudposse/terraform-aws-s3-log-storage/commit/32cfca282792756f0a9613e4bd4f7e3bf6484aca#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4)
> 
> Can you update your PR to account for these changes and also updates the examples accordingly?

There are several files and changes on that diff. It's not clear to me what changes you need me to do on my PR.

**Comment by woz5999** [comment]
The automated test are failing because `region` is no longer a valid variable: https://github.com/cloudposse/actions/runs/1150635697

i believe removing that var is the only change required at this time.

**Comment by woz5999** [comment]
@MartinCanovas thanks for the update, but you'll also need to remove region from `variables.tf` and from the examples in order for the automated tests to pass

**Comment by woz5999** [comment]
@MartinCanovas  you'll also need to update the examples for the automated tests to succeeed:
https://github.com/cloudposse/terraform-aws-lb-s3-bucket/blob/master/examples/complete/main.tf
https://github.com/cloudposse/terraform-aws-lb-s3-bucket/blob/master/examples/complete/variables.tf

File: main.tf [hunk 2]
```diff
@@ -38,13 +38,12 @@ data "aws_iam_policy_document" "default" {
 }
 
 module "s3_bucket" {
-  source                             = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.13.1"
+  source                             = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.14.0"
```
Comment (chancez) [inline main.tf:41]: 0.15.0 is out now

**Comment by joe-niland** [comment]
Hi @MartinCanovas could you please merge from master?

File: main.tf (hunk 2 above)
Comment (deadlysyn) [reply main.tf]: 0.16.0 is out now :-)

**Comment by MartinCanovas** [comment]
> This pull request is now in conflict. Could you fix it @MartinCanovas? 🙏

I see that several changes were made, and it appears this PR is no longer needed. Am I right?

**Comment by Gowiem** [comment]
@MartinCanovas I would assume it's no longer needed. Sorry it didn't work out. Please open a new PR if you've got any other issues.

**Pull request closed without merging**